crates/viewer/re_viewer/tests/app_kittest.rs

```diff
@@ -41,6 +41,19 @@ async fn settings_screen() {
     harness.snapshot("settings_screen");
 }
 
+/// Opens the Rerun menu without an active recording and snapshots the app.
+/// Tests that certain recording-related entries are disabled (e.g. save or close recording).
+#[tokio::test]
+async fn menu_without_recording() {
+    let mut harness = viewer_test_utils::viewer_harness();
+    harness.get_by_label("Menu").click();
+    harness.run_ok();
+    harness.snapshot_options(
+        "menu_without_recording",
+        &SnapshotOptions::new().threshold(2.0),
```

Comment:
I was fighting against a diff in two pixels' shade of orange on Mac vs Linux here.... 
<img width="767" height="188" alt="grafik" src="https://github.com/user-attachments/assets/5b4e710a-d667-4eb0-94fd-fc5a5110500f" />